tests/cases/unistore/get.js

```diff
@@ -0,0 +1,157 @@
+import QUnit from "qunit";
+import {get} from "../../../unistore";
+
+
+QUnit.module("unistore/get");
+
+
+QUnit.test(
+    "several test cases",
+    (assert) =>
+    {
+        const data = {
+            firstName: "John",
+            lastName: "Smith",
+            age: 25,
+            address: {
+                streetAddress: "21 2nd Street",
+                city: "New York",
+                state: "NY",
+                postalCode: "10021",
+            },
+            phoneNumber: [
+                {
+                    type: "home",
+                    number: "212 555-1234",
+                },
+                {
+                    type: "fax",
+                    number: "646 555-4567",
+                },
+            ],
+            gender: {
+                type: "male",
+            },
+        };
+
+        const cases = [
+            {
+                // multiple items
+                accessor: get`firstName,lastName`,
+                expected: {
+                    firstName: "John",
+                    lastName: "Smith",
+                },
+            },
+            {
+                // missing key
+                accessor: get`missing`,
+                expected: {
+                    missing: null,
+                },
+            },
+            {
+                // nested selector
+                accessor: get`address.city`,
+                expected: {
+                    city: "New York",
+                },
+            },
+            {
+                // aliasing
+                accessor: get`first_name:firstName`,
+                expected: {
+                    //eslint-disable-next-line
+                    first_name: "John",
+                },
+            },
+            {
+                // invalid nested selector
+                accessor: get`age.missing`,
+                expected: {
+                    missing: null,
+                },
+            },
+            {
+                // last access wins
+                accessor: get`name:firstName,name:lastName`,
+                expected: {
+                    name: "Smith",
+                },
+            },
+            {
+                // multiple matches
+                accessor: get`address.city,age`,
+                expected: {
+                    city: "New York",
+                    age: 25,
+                },
+            },
+            {
+                // renamed
+                accessor: get`prop1:address.city,prop2:age`,
+                expected: {
+                    prop1: "New York",
+                    prop2: 25,
+                },
+            },
+            {
+                // spaces don't matter (aliased)
+                accessor: get`prop1   :    address   .   city  ,   prop2  :   age`,
```

Comment:
Is having a `\n` a viable use-case so we can properly format complex retrievals like regular objects/arrays?

Reply:
This is already covered, as the whitespace stripping includes newlines as well:

![2018-05-15 at 12 09](https://user-images.githubusercontent.com/1032411/40050799-c8ee6450-5838-11e8-91e8-a143f43f3479.png)

I have added an explicit test case though. Also trailing commas are now allowed

Reply:
Great!